cfgov/unprocessed/root/data.json

```diff
@@ -261,9 +262,64 @@
       "distribution": [
         {
           "@type": "dcat:Distribution",
+          "description": "College Credit Card Agreements 2009-2019"
```

Comment:
JSON is finicky about missing commas, and I think the missing comma after this "description" line will be a troublemaker, as might the next 9 descriptions, which also don't have ending commas.

Reply:
I'm seeing missing commas on other description lines in your last commit as well.

Reply:
Thank you for pointing that out - I just went back and added commas to those lines

Reply:
Also might want to fix instances where "mediatype" is all lowercase, instead of camelcase "mediaType" – which appears to be what the validator is expecting.